packages/vue-client/src/components/boards/BadukBoardAbstract.vue

```diff
@@ -0,0 +1,77 @@
+<script setup lang="ts">
+import { Color } from "@ogfcommunity/variants-shared";
+import { PolygonalBoardHelperFunctions } from "@ogfcommunity/variants-shared/src/baduk/board/PolygonalBoardHelper";
+
+const helper = new PolygonalBoardHelperFunctions();
```

Comment:
Interesting. Is an instance of the helper injected by dependency injection? What happens if the helper has a non-empty constructor?

Thank you for teaching me :D

Reply:
> Is an instance of the helper injected by dependency injection?

In my example, there is no more `helper` - `CreatePolygonalBoard` just becomes a standalone function.  The reason this works is because both Min and CreatePolygonalBoard are functions that don't depend on the internal state of the object they are a part of.

> What happens if the helper has a non-empty constructor?

If the helper class had a constructor or properties, I think it would make more sense for it to remain a class.  But right now, the functions don't depend on the internal state of the class.